MARIE_SEQ2SEQ/deployment/flask/marie/templates/index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <title>Marie </title>
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="generator" content="GravCMS" />
+    <meta name="description"
+        content="Our Knowledge Graph (KG) services leverage advanced agent-based operation and ontology representations, to bring automation and semantic context to machine-assisted solutions." />
+    <link rel="icon" type="image/png" href="http://kg.cmclinnovations.com/user/themes/quark/images/favicon.png" />
+    <link rel="canonical" href="http://kg.cmclinnovations.com/explore/marie" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css"
+        integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link href="https://cdn.datatables.net/1.13.6/css/dataTables.bootstrap4.min.css" type="text/css" rel="stylesheet"/>
+    <link href="http://kg.cmclinnovations.com/user/plugins/form/assets/form-styles.css" type="text/css"
+        rel="stylesheet">
+    <link href="http://kg.cmclinnovations.com/user/themes/quark/css-compiled/spectre.min.css" type="text/css"
+        rel="stylesheet">
+    <link href="http://kg.cmclinnovations.com/user/themes/quark/css-compiled/theme.min.css" type="text/css"
+        rel="stylesheet">
+    <link href="http://kg.cmclinnovations.com/user/themes/quark/css/custom.css" type="text/css" rel="stylesheet">
+    <link href="http://kg.cmclinnovations.com/user/themes/quark/css/line-awesome.min.css" type="text/css"
```

Comment:
To clarify, do you mean that I should download these static files and put them in `deployment/flask/marie/static` so that they will be hosted by the flask app?

Reply:
Correct.

Reply:
I've replaced the links to externally hosted static resources with links to locally hosted resources.